### DISCUSSION TITLE
How to remove space between selection and "(" and ")"

### DISCUSSION THREAD

**Discussion by LukasPietzschmann** [kylechui/nvim-surround#148] [Q&A] [answered]
So imagine the following situation:
```
Old text           Command      New text
--------------------------------------------------
surr*ound_words     ysiw)    (surround_words)
```
But instead of `(surround_words)` I'm getting `( surround_words )` with an additional space between the parentheses and the text.

Is there a way to remove this space?

Edit: I didn't configure anything.

**Comment by kylechui** [comment, marked as answer]
Are you typing `ysiw)` exactly? There is a difference between `ysiw(` and `ysiw)`. In particular, the opening delimiter (e.g. `(` or `{`) will add whitespace, while the closing delimiter does not.

**Reply by LukasPietzschmann** [reply]
Ohhh no. I was typing `ysiw(` instead of `ysiw)`. Thanks a lot. I didn't think of that.
